blog/2023-01-30-awsbilling-slack-blog/index.md

```diff
@@ -192,53 +190,41 @@ We want to export our slack sink as an environmental variable so we can easily u
 export SLACK_SINK=$(kubectl get pods -n vanus | grep slack | awk '{ print $1 }')
 ```
 
-# Setting Up AWS Billing Source
-Just as we set up our Slack sink in a directory, we will also create a directory for our AWS Billing
-```shell
-mkdir billing
-```
-
-Change to the new directory and create two files; config.yml and secret.yml. The config.yml will take the configuration of our billing reports such as our target URL and the secret.yml will contain the access key and secret key obtained from your AWS console (IAM - Programmatic access)
-
-![img/img_6.png](img/img_6.png)
-
-Before updating our config.yml and secret.yml file. We need to create an [Eventbus](https://www.vanus.dev/introduction/concepts). An Eventbus represents a group of pipelines that receive and dispatch events. To create the eventbus,
+Before running our Amazon Billing Source, we will need to create an [Eventbus](https://www.vanus.dev/introduction/concepts). An Eventbus represents a group of pipelines that receive and dispatch events. To create the eventbus,
+Here, for simplicity, I have named the eventbus **billing**
 ```shell 
 vsctl eventbus create --name billing
 ```
 
-Here, for simplicity, I have named the eventbus **billing**
-
-Our target URL is the URL where we want to send cloud events to. For this, we will use our Vanus Gateway (192.168.49.2:30001). The target URL follows a specific pattern which is 
+# Setting Up AWS Billing Source
+Create a config file for Amazon Billing Source, don't forget the update the access key and secret key obtained from your AWS console (IAM - Programmatic access)
 
-```shell
-http://<ip_address>:<port>/gateway/<eventbus>
+```shell 
+cat << EOF > config.yml
+target: http://192.168.49.2:30001/gateway/billing
+secret:
+  access_key_id: AKIAIOSFODNN7EXAMPLE # <- Your access key
+  secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY <- Your secret key
```

Comment:
secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY #<- Your secret key

Reply:
Please make a suggestion in the future not a comment